massdash/ui/RawTargetedExtractionAnalysisUI.py

```diff
@@ -330,7 +330,9 @@ def show_extracted_one_d_plots(self, plot_container: st.container, chrom_plot_se
             A dictionary containing the plot objects for each file.
         """
         with plot_container:
-            if len(plot_dict[list(plot_dict.keys())[0]]) == 1:
+            if not plot_dict: # dictionary is empty
+                pass
```

Comment:
I don't think this necessarily means there are no plots because we can get into this loop if no plot is selected and the dataframe is selected. I think there is a warning somewhere else if nothing gets extracted. Currently get this if nothing found which I think is sufficient 
![image](https://github.com/Roestlab/massdash/assets/47336288/a0f3aadc-9877-4137-a688-d2f417e1959e)